File: projects/project01/rectangle_area.java
/*rectangle_area.java computes are of a rectangle

Input: The Length of a rectangle
       The Width of a rectangle
Precondition: both inputs are positive numbers
Output: The are of a rectangle

Created by: Mark Davis, CS 214, Calvin College
Date: February 9, 2016
Model Code: CS 214 Lab01
*********************************/

import java.util.Scanner; //include class for Scanner

public class rectangle_area
{
	/* function circleArea() computes a rectangle's area
	parameter: l, w, a double
	precondition: l and w are positive
	returns: area of rectangle
	*/	

	//main program
	public static void main(String[] args) 
	{

	//create a connection named keyboard to standard in
	Scanner keyboard = new Scanner(System.in);

	//prompt for length
	System.out.println("To compute the area, enter its LENGTH: ");	

	//get the number from the user
	double length = keyboard.nextDouble();

	//prompt for width
	System.out.println("Now enter its WIDTH: ");

	//get the number from the user
	double width = keyboard.nextDouble();

	//calculate the area
	double area = length * width;

	//output area
	System.out.println("The area is " + area);
	}
}

File: projects/project01/rectangle_area.rb
# rectangle_area.rb computes area of a rectangle
#
# Input: length of rectangle
#	 width of rectangle
#Precondition: the length and width are positive
#Output: the area of the rectangle
#
#Created by: mark Davis, CS 214, Calvin College
#Date: February 9, 2016
#Model Code: CS 214 Lab01
#
################################################

#function rectangle_area returns a rectangle's area, give length and width
#parameters: length, width
#precondition: l > 0, w > 0
#returns: area of a rectangle

def rectangleArea(length, width) #functin for computing area
	length * width #length multiplied by width
end

if __FILE__ == $0 
	print "Enter the length: " #prompt for length
	length = gets.chomp.to_f #store length value
	print "Enter the width: " #prompt for width
	width = gets.chomp.to_f #store width value
	print "Area is: " #output message for area
	puts rectangleArea(length, width) #call function to compute area
end


